content/docs/configuration.md

```diff
@@ -87,6 +87,12 @@ Github repository.
 (*list of strings*) A command which prints current upstream version (hint:
 `git describe`) (defaults to `git describe --abbrev=0 --tags --match '*'`).
 
+This option affects the behavior of following CLI commands:
+* build
```

Comment:
Sorry for not mentioning that before, but `build` only checkout dist-git branch and trigger build in koji. (No `srpm` build is done.)

```suggestion
```

Reply:
correct me if I am wrong, but according to this

https://github.com/packit/packit/blob/e7c970d589bda74a3ad7d77acdd72c12717acb4f/packit/api.py#L419-L420

 it can affect also `build` command

Reply:
Yes, you are right. Note that this is not a default behaviour but still influenced by that...